www/_layouts/contribute-help.html

```diff
@@ -0,0 +1,10 @@
+---
+layout: cordova
+no_code_highlight: true

```

Comment:
Small note: set `contribute_tab: true` here so that all the new contributing-related pages will have the tab highlighed.